### PR TITLE
feat(host_groups): vm-2001 and vm-2002 hostgroup added with parameters required by the vm

### PR DESCRIPTION
**Pull request by smirta**

### CODE REVIEW

File: roles/foreman/tasks/host_groups.yaml [hunk 1]
```diff
@@ -167,6 +167,110 @@
...
+          - name: firewall
+            parameter_type: yaml
+            value:
+              - service: http-alt
+                port: 8080/tcp
+                state: present
+                permanent: true
+              - service: https-alt
+                port: 8443/tcp
+                state: present
```
Comment (hairmare) [inline roles/foreman/tasks/host_groups.yaml:192]: isn't it service or port on these?

did you consider putting some of them into a parent group?

File: roles/foreman/tasks/host_groups.yaml [hunk 2]
```diff
@@ -167,6 +167,110 @@
...
+              - state: started
+                kube_file_content:
+                  apiVersion: v1
+                  kind: Pod
+                  metadata:
+                    name: revproxy
+                  spec:
+                    containers:
+                    - name: revproxy
+                      image: ghcr.io/radiorabe/httpd:0.3.0
```
Comment (hairmare) [inline roles/foreman/tasks/host_groups.yaml:234]:
```suggestion
                      image: ghcr.io/radiorabe/httpd:0.5.0
```

File: roles/foreman/tasks/host_groups.yaml [hunk 3]
```diff
@@ -176,3 +280,107 @@
...
```
Comment (hairmare) [inline roles/foreman/tasks/host_groups.yaml:347]:
```suggestion
                      image: ghcr.io/radiorabe/httpd:0.5.0
```

**Review by hairmare — COMMENTED** [review]
I'm going to apply the AlmaLinux 9.5 update via this change:

File: roles/foreman/tasks/host_groups.yaml (hunk 1 above)
Comment (smirta) [reply roles/foreman/tasks/host_groups.yaml:192]: I thought it's prettier to add the ports `8080/tcp` and `8443/tcp` as services `http-alt` and `https-alt`. I did not consider to putting them into a parent group. We do not have a use-case for them on other hosts afaik

File: roles/foreman/tasks/host_groups.yaml [hunk 4]
```diff
@@ -167,6 +167,110 @@
         subnet: dmz
         compute_resource: server-008.dmz-admin.int.rabe
         compute_profile: 1-Small	
+      - name: vm-2001.dmz.int.rabe.ch
+        description: >
+          AlmaLinux 9 DMZ virtual machine vm-2001
+           for running reverse-proxy container
+        parent: >
+          RaBe Core/RaBe Base/EL9/AlmaLinux 9/
+          AlmaLinux 9 DMZ server-009 Vms
```
Comment (hairmare) [inline roles/foreman/tasks/host_groups.yaml:176]:
```suggestion
        parent: RaBe Core/RaBe Base/EL9/AlmaLinux 9/AlmaLinux 9 DMZ server-009 Vms
```

File: roles/foreman/tasks/host_groups.yaml [hunk 5]
```diff
@@ -176,3 +280,107 @@
         subnet: dmz
         compute_resource: server-009.dmz-admin.int.rabe
         compute_profile: 1-Small	
+      - name: vm-2002.dmz.int.rabe.ch
+        description: >
+          AlmaLinux 9 DMZ virtual machine vm-2002
+           for running reverse-proxy container
+        parent: >
+          RaBe Core/RaBe Base/EL9/AlmaLinux 9/
+          AlmaLinux 9 DMZ server-009 Vms
```
Comment (hairmare) [inline roles/foreman/tasks/host_groups.yaml:289]:
```suggestion
        parent: RaBe Core/RaBe Base/EL9/AlmaLinux 9/AlmaLinux 9 DMZ server-009 Vms
```

**Review by hairmare — COMMENTED** [review]
shouldn't parent be on a single line?

File: roles/foreman/tasks/host_groups.yaml [hunk 6]
```diff
@@ -167,6 +167,110 @@
...
+            value: true
+          - name: podman_firewall
+            parameter_type: yaml
+            value:
+              - port: 8080/tcp
+                state: enabled
+              - port: 8090/tcp
+                state: enabled
+              - port: 8443/tcp
+                state: enabled
```
Comment (hairmare) [inline roles/foreman/tasks/host_groups.yaml:221]: Shouldn't these use the defined services `http-alt` and `https-alt`?

Comment (hairmare) [reply roles/foreman/tasks/host_groups.yaml:268]: We can change this later, both ways should work

**Pull request merged**